sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_dsl_pipeline.py

```diff
@@ -3261,3 +3275,18 @@ def pipeline_func():
             node_compute = rest_obj.properties.jobs[node_name]["computeId"]
             assert is_singularity_id_for_resource(node_compute)
             assert node_compute.endswith(singularity_vc.name)
+
+    def test_assign_value_to_unknow_filed(self, client: MLClient):
+        path = "./tests/test_configs/components/helloworld_component.yml"
+        component_func = load_component(source=path)
+        component_func.tags = {}
```

Comment:
maybe add a comment here to explain why we use this workaround?

Reply:
done